Doc/library/subprocess.rst

```diff
@@ -723,6 +723,11 @@ Instances of the :class:`Popen` class have the following methods:
    On Windows :meth:`kill` is an alias for :meth:`terminate`.
 
 
+.. method:: Popen.kill_on_error()
+
+   Kills the child when a python error raised.
```

Comment:
When a python error is raised from where and when?

Reply:
Hi, Gregory, thank for your elaborative review:).
one practical case would be found in https://bugs.python.org/issue25122(due to some race condition on FreeBSD) and i updated the subprocess.rst too.